email_notification/sender/email_notification_builder.py

```diff
@@ -11,28 +11,44 @@
 
 from applications.models import CUSTOMER_TYPES
 from email_notification.models import EmailTemplate
+from email_notification.templatetags.email_template_filters import format_currency
 from reservations.models import Reservation
 from tilavarauspalvelu.utils.commons import LANGUAGES
 
+FILTERS_MAP = {"currency": format_currency}
+
 
 def get_sandboxed_environment() -> SandboxedEnvironment:
     env = SandboxedEnvironment()
+
+    for fil, func in FILTERS_MAP.items():
+        env.filters[fil] = func
+
     return env
 
 
 class EmailTemplateValidator:
     @property
     def bracket_lookup(self):
-        return re.compile(r"{{ *(\w+) *}}")
+        return re.compile(r"{{ *(\w+) \| *(\w+) *}}|{{ *(\w+) *}}")
```

Comment:
Have to admit I have hard time understand what this regex should match 😄

Reply:
words within {{}} and {{ | }}